mso/resource_mso_schema_site_anp_epg_domain.go

```diff
@@ -152,31 +190,46 @@ func resourceMSOSchemaSiteAnpEpgDomainImport(d *schema.ResourceData, m interface
 	if err != nil {
 		return nil, fmt.Errorf("No Sites found")
 	}
-
-	stateSite := get_attribute[2]
+	templateInfo := regexp.MustCompile("-").Split(get_attribute[2], 2)
```

Comment:
I am not clear on what the regexp do in this line.

Reply:
get_attribute[2] = {site_id}-{template_name}
It gets the template name from above.